pkg/props/page.go

```diff
@@ -39,6 +39,10 @@ type PageNumber struct {
 	Pattern string
 	// Place defines where the page count component will be placed.
 	Place Place
+	// MarginTop is the margin top of the page count component.
+	MarginTop float64
+	// MarginRight is the margin right of the page count component.
+	MarginLeft float64
```

Comment:
It this the same value that we have inside cfg.Margins? That we compute in builder... If it is, we should pass this in props, because it will get strange...It this is the same margin top, we should get from config...

Reply:
IIUC, pagenumber is a kind of special text, we should not inherit from page config.

the original idea is make footRow and pageNumber work together with better display.